background/constants/coin-types.ts

```diff
@@ -2,9 +2,10 @@
 // data but is important, it SHOULD NOT be imported from an external package.
 
 export const COIN_TYPES_BY_ASSET_SYMBOL = {
+  QI: 1, // FIXME
```

Comment:
Currently, the coin ID for QUAI is set to 994. What is the correct coin ID for the Qi token that we should use in this context?

Reply:
It is 969. I updated it.